### PR TITLE
Add a script to update version number

### PR DESCRIPTION
**Pull request by ashie**
# Which issue(s) this PR fixes:

N/A.

# What this PR does / why we need it:

Currently we need to update version number of each modules respectively, it's too bother.
It's convenient to add a script to update all version numbers.

In addition, we should apply same version number to each modules in future releases to avoid confusion.

It's based on @yashirot's work in https://gitlab.com/clear-code/browserselector/-/tree/e64d416a8fff577068783d2a9782a9caebdb5c8f/script

# How to verify the fixed issue:

Run the script then check `git diff` to verify that the script expectedly replace version numbers.

## The steps to verify:

1. Open PowerShell window (or something similar) at the script folder
2. Run ` powershell.exe -ExecutionPolicy Bypass -file .\ThinBridgeVersionUp.ps1 4.2.1.0`
3. Check `git diff`
4. Run unit test: `powershell.exe -ExecutionPolicy Bypass Invoke-Pester`

## Expected result:

> 3. Check `git diff`

Version numbers are replaced with `4.2.1.0`

> 4. Run unit test: `powershell.exe -ExecutionPolicy Bypass Invoke-Pester`

Pass all tests

### CODE REVIEW

**Comment by ashie** [comment]
This PR also modify newline characters in ThinBridge*.iss files from `LF` to `CR+LF` to minimize the result of the following procedure:

> 3. Check `git diff`

**Comment by ashie** [comment]
I've confirmed that the installer is successfully created with updated code by this script.
https://github.com/ThinBridge/ThinBridge/tree/v4.2.1.0-rc1

**Comment by ashie** [comment]
修正大きく見えますが、script/fixtures以下は既存のものをコピーしてバージョン番号をテスト用に置き換えたのみ、ThinBridge*.issは改行コードが`LF`だったので`CR+LF`に修正しただけです。

File: script/ThinBridgeVersionUp.ps1 [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+		$replaced_content = (Get-Content -Path $item -Encoding $encoding) |
+			ForEach-Object {
+				if ($_ -like "*version*") {
+					$_ -Replace $VERSION_PERIOD, $new_version
+				} else {
+					$_
+				}
+			} |
+			ForEach-Object {
+				if (($_ -like "*version*") -and ($_ -notlike "*IDC_STATIC_VER*")) {
```
Comment (ashie) [inline script/ThinBridgeVersionUp.ps1:62]: 元のコードのままだとThinBridgeでは非該当箇所（ウインドウサイズ等）が大量に置換されてしまうので`($_ -like "*version*")`でマッチングさせるようにした。
一箇所、バージョン番号とウインドウサイズ?の両方が記載されている行があり、どうしてもうまく汎化した除外条件を書けなかったので、該当箇所の除外専用に`($_ -notlike "*IDC_STATIC_VER*")`も入れた。

Comment (yashirot) [reply script/ThinBridgeVersionUp.ps1:62]: 承知しました。

File: README.md [hunk 2]
```diff
@@ -29,21 +29,24 @@ ThinBridgeはエンタープライズ環境向けのブラウザ切替えソリ
...
-    - ThinBridgeSetupX86.iss
- 4. 次のコマンドでタグを打ってプッシュする
-    ```sh
-    $ make release
+以下、リリースするバージョンを4.2.1.0と仮定して説明する。
+
+ 1. PowerShellウインドウ等を開き、script/ThinBridgeVersionUp.ps1 を使ってインストーラおよび各モジュールのバージョンを更新する
+    ```console
+    > cd script
+    > powershell.exe -ExecutionPolicy Bypass -file .\ThinBridgeVersionUp.ps1 4.2.1.0
```
[the remark this answers is not in the document]
Comment (ashie) [reply README.md]: You're right! Thanks for your comment!
I've applied your suggestion.

**Pull request merged**